jina/executors/crafters/image/flip.py

```diff
@@ -0,0 +1,66 @@
+from typing import Dict
+
+import numpy as np
+
+from .. import BaseCrafter
+from .helper import _restore_channel_axis, _load_image
+
+
+class ImageHorizontalFlipper(BaseCrafter):
```

Comment:
There is a lot of shared logic in these 2 classes; wouldn't it be better to have a general ImageaFlipper that is generally specialized implementing different flip functions?

Reply:
Yes, you are right. I implemented it as PyTorch does for RandomHorizontalFlip and RandomVerticalFlip. I will update it.